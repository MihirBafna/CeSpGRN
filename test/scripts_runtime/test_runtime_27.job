#!/bin/bash
#SBATCH -J test_runtime_27
#SBATCH -N 1
#SBATCH -p RM-shared
#SBATCH -t 2:00:00
#SBATCH -o test_runtime_27.out 
#SBATCH -e test_runtime_27.err
#SBATCH --mail-user=zzhang834@gatech.edu
#SBATCH --mail-type=ALL

module load anaconda3
module load cuda/11.1.1
conda activate myenv

cd ${PROJECT}/DynGRN/test/scripts_runtime

# 1st argument: (gene,tf) pair, can be 0,1,2,3,4
# 2nd argument: seed, 0,1,2
gene_tf=2
seed=2
python test_runtime.py ${gene_tf} ${seed}
