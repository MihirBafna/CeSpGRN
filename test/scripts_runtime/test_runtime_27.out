ngenes: 100, seed: 2
number of nearest neighbor: 6
number of nearest neighbor: 7
final number of nearest neighbor (make connected): 7
number of neighbor being considered: 30
time calculating the kernel function: 933.67 sec
time cost (estimating empirical covariance matrix): 933.68 sec
start running batch 0
n_iter: 500, duality gap: 5.4192e-08, primal residual: 5.0828e-07, dual residual: 1.083951e-06
Batch loss: loss1: 8607.76660, loss2: 29648.26367, loss3: 0.00000
Finished running batch 0, running time: 208.09 sec
start running batch 1
n_iter: 500, duality gap: 3.7555e-08, primal residual: 3.4561e-07, dual residual: 9.816630e-07
Batch loss: loss1: 8929.37402, loss2: 27613.76562, loss3: 0.00000
Finished running batch 1, running time: 208.24 sec
start running batch 2
n_iter: 500, duality gap: 3.6015e-08, primal residual: 3.2532e-07, dual residual: 9.908335e-07
Batch loss: loss1: 9047.91602, loss2: 26827.62500, loss3: 0.00000
Finished running batch 2, running time: 201.40 sec
start running batch 3
n_iter: 500, duality gap: 3.7030e-08, primal residual: 2.9194e-07, dual residual: 9.987883e-07
Batch loss: loss1: 8917.98438, loss2: 27915.87500, loss3: 0.00000
Finished running batch 3, running time: 206.78 sec
start running batch 4
n_iter: 500, duality gap: 3.0995e-08, primal residual: 3.4955e-07, dual residual: 9.785934e-07
Batch loss: loss1: 8596.59863, loss2: 30328.21875, loss3: 0.00000
Finished running batch 4, running time: 213.51 sec
start running batch 5
n_iter: 500, duality gap: 4.5230e-08, primal residual: 4.5916e-07, dual residual: 1.002358e-06
Batch loss: loss1: 8101.12451, loss2: 33554.66797, loss3: 0.00000
Finished running batch 5, running time: 217.38 sec
start running batch 6
n_iter: 500, duality gap: 3.2391e-08, primal residual: 3.1303e-07, dual residual: 9.177203e-07
Batch loss: loss1: 9031.23145, loss2: 27077.99414, loss3: 0.00000
Finished running batch 6, running time: 205.92 sec
start running batch 7
n_iter: 500, duality gap: 3.2219e-08, primal residual: 3.2707e-07, dual residual: 9.692865e-07
Batch loss: loss1: 8899.81152, loss2: 28259.81445, loss3: 0.00000
Finished running batch 7, running time: 206.09 sec
start running batch 8
n_iter: 500, duality gap: 3.1328e-08, primal residual: 3.3172e-07, dual residual: 9.196021e-07
Batch loss: loss1: 8583.05273, loss2: 30675.57422, loss3: 0.00000
Finished running batch 8, running time: 206.42 sec
start running batch 9
n_iter: 500, duality gap: 4.8935e-08, primal residual: 4.0385e-07, dual residual: 9.947178e-07
Batch loss: loss1: 8193.79688, loss2: 33302.32031, loss3: 0.00000
Finished running batch 9, running time: 207.99 sec
time cost (estimating theta): 2095.80 sec
